Library/Homebrew/cli/parser.rb

```diff
@@ -24,6 +24,9 @@ class Parser
       }.freeze, T::Hash[Symbol, String])
       private_constant :ArgType, :HIDDEN_DESC_PLACEHOLDER, :SYMBOL_TO_USAGE_MAPPING
 
+      sig { returns(Args) }
+      attr_reader :args
```

Comment:
i found this to be preferred to doing leaps with `@args` in `ParserLinux` (and args are frozen after parsing).